### DISCUSSION TITLE
react table - get selected rows with pagination

### DISCUSSION THREAD

**Discussion by vikrantvi** [TanStack/table#3463] [Q&A]
Hi all, how do i get all selected rows with pagination. on row selection it gives me like {0:true} its row index. how i get row id in place of row index.

[https://codesandbox.io/s/github/tannerlinsley/react-table/tree/master/examples/kitchen-sink](https://codesandbox.io/s/github/tannerlinsley/react-table/tree/master/examples/kitchen-sink)

**Comment by narendradharajiya** [comment]
add `selectedFlatRows` for getting selected rows with pagination.
